libpod/runtime_ctr.go

```diff
@@ -246,7 +246,15 @@ func (r *Runtime) removeContainer(ctx context.Context, c *Container, force bool)
 	}
 
 	if c.state.State == ContainerStatePaused {
-		return errors.Wrapf(ErrCtrStateInvalid, "container %s is paused, cannot remove until unpaused", c.ID())
+		if !force {
+			return errors.Wrapf(ErrCtrStateInvalid, "container %s is paused, cannot remove until unpaused", c.ID())
+		}
+		if err := c.runtime.ociRuntime.killContainer(c, 9); err != nil {
```

Comment:
Move all of this down into the next block - we need the sync to pick up container state at the end

Reply:
Specifically make this part of the existing "stop container if force set" code just below this

Reply:
per our irc conversation, just adding a sync